### PR TITLE
Arkansas Total Exemptions

### LINKED ISSUE #3040: Arkansas Total Exemptions
Arkansas Total Exemptions from Income Tax

Ref: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=10
Page 10~11

Ref: Arkansas 2022 Individual Income Tax Forms and Instructions
Page 9~10

### PR DESCRIPTION
**Pull request by shuyu7zhan**
Fixes #3040

### Summary
📄🏷️➕

This pull request adds parameters and variables for the exemptions that reduce taxable income for Arkansas state income tax. It creates a new parameter file `exemptions.yaml` and a new variable file `ar_total_exemptions.py` under the `ar` state folder.

> _`Exemptions` added_
> _Arkansas tax parameters_
> _Autumn leaves fall fast_

### Walkthrough
* Add a new parameter file for Arkansas income tax exemptions ([link](https://github.com/PolicyEngine/policyengine-us/pull/3053/files?diff=unified&w=0#diff-c32117c9a8a6ee7356765131829f47c5a9e7bbc5d2c2279bd1640cdb8209a7a2R1-R28))
* Add a new variable file for the total amount of exemptions for Arkansas income tax ([link](https://github.com/PolicyEngine/policyengine-us/pull/3053/files?diff=unified&w=0#diff-5ff2d27b63b2fe9cdfc18cb727f4bc88a6b0dfe127e219b68d6463000dc5bbf3R1-R12))

### CODE REVIEW

**Comment by shuyu7zhan** [comment]
Exemption Item 10, 12 & 13 remain to be further checked.

File: policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml:1]: need changelog_entry

File: policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+description: Arkansas counts these exemptions when calculating its income tax.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml:1]:
```suggestion
description: Arkansas account for these exemptions when calculating the state adjusted gross income.
```

File: policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    # Item 15 "PSLF"
+
+metadata:
+  unit: list
+  label: Arkansas exemptions
+  reference:
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions
+      href:https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=10
+    - title: Arkansas 2021 Individual Income Tax Forms and Instructions
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2021_AR1000F_and_AR1000NR_Instructions.pdf
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml:25]: need page

File: policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    - military_basic_pay # Item 11
+    # Item 14 Community Match Rural Physician Recruitment Program
+    # Item 15 "PSLF"
+
+metadata:
+  unit: list
+  label: Arkansas exemptions
+  reference:
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions
+      href:https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=10
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml:23]:
```suggestion
      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=10
```

File: policyengine_us/variables/gov/states/ar/tax/income/ar_total_exemptions.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+from policyengine_us.model_api import *
+
+
+class ar_total_exemptions(Variable):
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ar/tax/income/ar_total_exemptions.py:4]:
```suggestion
class ar_exemptions(Variable):
```

File: policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml
Comment (shuyu7zhan) [reply policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml:1]: > need changelog_entry

OK added. Long time no create an issue, totally forgotten

File: policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml [hunk 5]
```diff
@@ -0,0 +1,28 @@
+description: Arkansas accounts for these exemptions when calculating the state adjusted gross income.
+values:
+  2021-01-01:
+    - military_retirement_pay # Item 1
+    # Item 2&3 Life insurance policy
+    - child_support_received # Item 4
+    # Item 5 Gifts, inheritances, bequests, or devises
+    # Item 6 Scholarships, fellowships, and grants 
+    - us_govt_interest # Item 7
+    - workers_compensation # Item 8
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml:10]: need to check what VA benefits - I think its veteran related

File: policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    - social_security # Item 8
+    # Item 9 housing allowance paid to church
+    - disability_benefits # Item 10
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml:13]: what does the "may" relate to?

File: policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml (hunk 5 above)
Comment (shuyu7zhan) [reply policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml:10]: Found "veterans_benefits". Does that fit?

> need to check what VA benefits - I think its veteran related

File: policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml (hunk 6 above)
Comment (shuyu7zhan) [reply policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml:13]: > what does the "may" relate to?

Not quite understand.

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml:13]: image

Comment (shuyu7zhan) [reply policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml:13]: > image

https://answerconnect.cch.com/federal/arp1209013e2c83dc42ad/irc/current/compensation-for-injuries-or-sickness

There might be some restrictions or specified circumstances when applying the disability exemption? Not sure about how to code that

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ar/tax/income/exemptions.yaml:13]: if it is a federal dependency then we can just add a comment in the list saying that

**Comment by MaxGhenis** [comment]
Please remove the issue number from the title and move it to the description to link it

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]
For posterity, Arkansas uses the term exemptions to refer to what most states call subtractions. The tax form calls it exempt income sources while the legal code uses the term exemptions. This is not about personal or dependent exemptions.

**Pull request merged**